.gitignore

```diff
@@ -102,3 +102,146 @@ dist
 
 # TernJS port file
 .tern-port
+
```

Comment:
I feel we should just ignore the ones we generate.

Reply:
Yeah, this `.gitignore` is absolutely huge. I think we should only ignore stuff related to this project that is common to every checkout (like `__pycache__` and `.pyc`) and push devs to use `.git/info/exclude` for their IDEs/editors or specific setup.

Reply:
Removed almost everything there and kept only what we need